### DISCUSSION TITLE
[Project Beta] Delete useless project

### DISCUSSION THREAD

**Discussion by fhigher** [community/community#14366] [Other Feature Feedback, Questions, & Ideas]
[Project Beta] How to delete project that you no longer want to use？Not just closing the project.

**Comment by etsploit** [comment]
Yeah I want to delete a board it's getting on my nerves that it's there

**Comment by suryanshmak** [comment]


**Reply by etsploit** [reply]
Project board not repo

**Comment by suryanshmak** [comment]


**Reply by etsploit** [reply]
I'm telling you that this does not apply to beta projects.

**Comment by queenofcorgis** [comment]
Thanks for this feedback, I've added some labeling so our team can easily find this. They'll comment here if and when they have any updates.
